views/js/previewer/plugins/content/scoringHighlighter.js

```diff
@@ -89,6 +70,58 @@ define([
                 })
             );
 
+            /**
+             * Gets the renges of the selection
+             * 
+             * @param {*} selection 
+             */
+            function getAllRanges(selection) {
+                const allRanges = [];
+
+                for (let i = 0; i < selection.rangeCount; i++) {
+                    allRanges.push(selection.getRangeAt(i));
+                }
+                return allRanges;
+            }
+
+            /**
+             * Erases highlights and updates highlightIndex
+             * 
+             * @param {Event} e - Click event
+             */
+            function clearHighlightAndSave(e) {
+                highlighter.clearSingleHighlight(e)
```

Comment:
Missed `;`

Reply:
I am so used to prettier helping me and its not working in tao 😢